mllib/src/main/scala/org/apache/spark/ml/regression/AFTSurvivalRegression.scala

```diff
@@ -421,9 +421,17 @@ class AFTSurvivalRegressionModel private[ml] (
     }
 
     if (hasQuantilesCol) {
-      val predictQuantilesUDF = udf { features: Vector => predictQuantiles(features)}
+      val baseQuantiles = $(quantileProbabilities)
+        .map(q => math.exp(math.log(-math.log1p(-q)) * scale))
+      val lambdaCol = if ($(predictionCol).nonEmpty) {
```

Comment:
I think you're probably right here but why is lambdaCol the first predictionCol here?

Reply:
if ($(predictionCol).nonEmpty), then `predictionColumns.head` is `predictUDF(col($(featuresCol))).as($(predictionCol), outputSchema($(predictionCol)).metadata)`, the predictionCol.